tutorials/scripting/evaluating_expressions.rst

```diff
@@ -198,7 +198,7 @@ The output from the script will be::
 Built-in functions
 ------------------
 
-Most methods available in the :ref:`class_@GDScript` scope are available in the
+All methods in the :ref:`class_@GlobalScope` are available in the
```

Comment:
```suggestion
All methods in :ref:`class_@GlobalScope` are available in the
```
I'd that without the "scope" after it is more correct to just say "in X" as it is a name itself

Reply:
True, thanks for the tip. I like the second version better.